### PR TITLE
Comparison events

### PR DESCRIPTION
**Pull request by Anna4142**

### CODE REVIEW

**Comment by Anna4142** [comment]
i need to add the ground truth data dile explicietly in the folder where the results are saved.any way to add it at the time the directory is created?

File: Arduino_related_code/ArduinoDigital.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 def openComPort(portname):
     global comport;
     comport = serial.Serial(portname, 9600)
-    time.sleep(2)                # Arduino resets itself when the port is opened. Give it time to complete.
+    time.sleep(2)
```
Comment (MickyHoltzman) [inline Arduino_related_code/ArduinoDigital.py]: Why did you get rid of my comment, you did not like it?

File: Arduino_related_code/ValveCaliberationTest.py [hunk 2]
```diff
@@ -45,8 +45,8 @@ def calibrate_valve(pin, duration_ms,opening_number, arduino):
 # Button to start calibration
 calibrate_button = tk.Button(root, text="Start Calibration",
                              command=lambda: calibrate_valve(int(pin_entry.get()),
-                                                               int(duration_entry.get()),int(OpeningNumber_entry.get()),
-                                                             int(arduino)))
+                                                               int(duration_entry.get()),int(OpeningNumber_entry.get()),arduino))
```
Comment (MickyHoltzman) [inline Arduino_related_code/ValveCaliberationTest.py]: there is something wrong here. Arduino is not a class any more. there should not be a need to pass the Arduino instance.

File: Arduino_related_code/arduinovalvecontrol.txt [hunk 3]
```diff
@@ -1,117 +1,210 @@
 // Serial port constants
 const int baud_rate = 9600;
 
-// Control constants 
-const int ONBOARD_LED          = 13;    // digital out to drive the on board LED for heart bit
-const int SLOW_HEART_BIT_DELAY = 500;  // Delay time for slow rate heartbit 
-const int FAST_HEART_BIT_DELAY = 200;  // delay time for the fast rate heartbit
-const int NUM_DIGITAL_CHANNEL  = 16;    // number of digital channels. 
+// Control constants
```
Comment (MickyHoltzman) [inline Arduino_related_code/arduinovalvecontrol.txt]: I thought we agreed this file is not needed any more.

File: Experiment_Launcher_code/ModuleConfiguration.py [hunk 4]
```diff
@@ -1,4 +1,4 @@
-__USE_VIDEO_SIM = False
-__USE_VIDEO_STUB = False
-__USE_ARDUINO_SIM = False
+__USE_VIDEO_SIM =True
```
Comment (MickyHoltzman) [inline Experiment_Launcher_code/ModuleConfiguration.py]: I think these should be false for a "production system"

File: Experiment_Launcher_code/ProjectDirectory.txt [hunk 5]
```diff
@@ -1 +1 @@
-C:/Users/EngelHardBlab.MEDICINE/Desktop/experimentfolder/PILOT_RESULTS_2
\ No newline at end of file
+C:/Users/anush/Downloads/Experiment_Folder
```
Comment (MickyHoltzman) [inline Experiment_Launcher_code/ProjectDirectory.txt:1]: Are you sure you want to change the project directory file?

File: Experiment_Launcher_code/experimentgui.py [hunk 6]
```diff
@@ -261,6 +263,40 @@ def validate_inputs(self):
 
         return True
 
+    def update_opponent_options(self):
```
Comment (MickyHoltzman) [inline Experiment_Launcher_code/experimentgui.py:266]: isn't this what you wanted me to do? So, is it done?

**Review by MickyHoltzman — COMMENTED** [review]
Some comments, Please take a look

**Review by MickyHoltzman — APPROVED** [review]
go for it

**Pull request merged**